Makefile

```diff
@@ -24,7 +27,6 @@ check-license:
 	./scripts/check_license.sh
 
 container:
```

Comment:
That was the intention before, however unfortunately minikube does not mount directories from the host machine correctly on linux, therefore what linux users have to do is build the binary with local docker host and then use that binary within minikube.